app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -158,6 +158,7 @@
         with_border: !omit_footer_border,
         navigation: omit_footer_navigation ? nil : layout_helper.footer_navigation,
         meta: layout_helper.footer_meta,
+        ga4_index_total: layout_helper.get_footer_length
```

Comment:
This grabs the `@footer_length` value from `public_layout_helper.rb` so that the OGL and Crown Copyright link can access it.